mailgun/views.py

```diff
@@ -26,5 +26,5 @@ def log_post_data(request):
     :param request:
     :return HttpResponse:
     """
-    logger.info(u'[MAILGUN EVENT] %s', json.dumps(request.POST, indent=4, sort_keys=True))
+    logger.info(u'[MAILGUN EVENT] %s', json.dumps(request.POST, separators=(',', ': '), sort_keys=True))
```

Comment:
I think you only need to specify the separators to avoid whitespace when you're also specifying the indent.